vignettes/estimate_relation.Rmd

```diff
@@ -75,8 +73,8 @@ approximations using
 ### Polynomial
 
 ```{r message=FALSE, warning=FALSE, eval=FALSE}
-glm(Sepal.Length ~ poly(Sepal.Width, 2), data = iris) %>%
-  modelbased::estimate_relation(length = 50) %>%
+glm(Sepal.Length ~ poly(Sepal.Width, 2), data = iris) |>
```

Comment:
Don't use the new `|>` pipe operator because, otherwise, we will have to bump the needed R version to R 4.1.

We currently support all R versions > 3.4.

Reply:
is it ok to have magrittr in suggests then?

Reply:
AFAIK, the _poorman_ package also provides the pipe!

Reply:
ah right